.github/workflows/test-and-release.yaml

```diff
@@ -110,11 +111,11 @@ jobs:
         with:
           inlineScript: |
             az keyvault secret set --vault-name ${keyVaultName} --name unwiredlabsApiKey --value my-secret
-            openssl ecparam -name secp256k1 -genkey > nrfcloudcelllocationservice.key
+            openssl ecparam -name prime256v1 -genkey > nrfcloudcelllocationservice.key
```

Comment:
Unrelated change.

Reply:
So, it's unrelated but required for CI to pass?

Reply:
Yes